### PR TITLE
Fix (Journald logging) localStreaming typo.

### PR DESCRIPTION
**Pull request by jollinshead**

### CODE REVIEW

**Comment by jollinshead** [comment]
This is just to tidy up the localStreaming feature of Journald logging in anticipation of an upcoming pull-request.

**Comment by mumoshu** [comment]
@jollinshead Thanks for your continuous efforts on this area 👍

**Comment by jollinshead** [comment]
@mumoshu Apologies for sneaking in a few more commits.
For now, I have made all the changes I had intended.

File: core/root/cluster.go [hunk 1]
```diff
@@ -309,9 +309,9 @@ func (c clusterImpl) Update() (string, error) {
 
 	if c.controlPlane.CloudWatchLogging.Enabled && c.controlPlane.CloudWatchLogging.LocalStreaming.Enabled {
 		// Return Journald logs in a separate GoRoutine
-		quit := make(chan bool)
-		defer func() { quit <- true }()
-		go streamJournaldLogs(c, quit)
+		q := make(chan struct{})
```
Comment (mumoshu) [inline core/root/cluster.go]: Just for my education but why did you migrate from `chan bool` to `chan struct{}`? Were there any pros/cons?

Comment (jollinshead) [inline core/root/cluster.go]: Pro: 1 less byte of memory :smile: https://stackoverflow.com/questions/22770114/any-difference-in-using-an-empty-interface-or-an-empty-struct-as-a-maps-value-i

**Comment by mumoshu** [comment]
@jollinshead Thank you as always for your contribution 👍

**Pull request merged**